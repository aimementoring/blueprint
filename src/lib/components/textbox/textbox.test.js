import React from 'react';
import renderer from 'react-test-renderer';
import TextBox from './textbox';

describe('TextBox', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(
        <TextBox
          placeholder="Write area"
          name="textbox"
          required
          className="myClass"
          onChangeFunction={value => {}}
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
