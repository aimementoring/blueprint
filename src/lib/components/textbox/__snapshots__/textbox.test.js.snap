// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextBox renders properly 1`] = `
<div
  className="theme-purple"
>
  <div
    className="theme-purple"
  >
    <div
      className="inputWrapper "
    >
      <textarea
        className="input textarea myClass
              false"
        name="textbox"
        onChange={[Function]}
        placeholder="Write area"
        required={true}
        value=""
      />
    </div>
  </div>
  
</div>
`;
